<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scenarios Catalog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_title">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Project Configuration: &lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;QTRANUS&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Scenarios&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_help">
       <property name="text">
        <string notr="true">Help</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTreeView" name="scenarios_tree"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="add">
         <property name="text">
          <string>Add Scenario</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/add-scenario.svg</normaloff>icons/add-scenario.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="remove">
         <property name="text">
          <string>Remove Scenario</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/remove-scenario.svg</normaloff>icons/remove-scenario.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="copy">
         <property name="text">
          <string>Copy Scenario</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/copy-scenario.svg</normaloff>icons/copy-scenario.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="paste">
         <property name="text">
          <string>Paste Scenario</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/paste-scenario.svg</normaloff>icons/paste-scenario.svg</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btn_help</tabstop>
  <tabstop>scenarios_tree</tabstop>
  <tabstop>remove</tabstop>
  <tabstop>copy</tabstop>
  <tabstop>paste</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
